js/src/dropdown.js

```diff
@@ -192,10 +187,10 @@ const Dropdown = (($) => {
           continue
         }
 
-        // remove backdrop only if the dropdown menu will be hidden
-        const backdrop = $(parent).find(Selector.BACKDROP)[0]
-        if (backdrop) {
-          backdrop.parentNode.removeChild(backdrop)
+        // if this is a touch-enabled device we remove the extra
+        // empty mouseover listeners we added for iOS support
+        if ('ontouchstart' in document.documentElement) {
+          $('body').children().off('mouseover', $.noop)
```

Comment:
Here `$.noop` will continue to be called because the second parameter of ``off()`` is a selector see https://api.jquery.com/off/#off-events-selector

To check what I said you can test that : 
```javascript
var lala = function () { console.log('lala') }
var toto = function () { console.log('toto') }
$('body').children().on('mouseover', lala)
$('body').children().on('mouseover', toto)
```
And after try to do this : `$('body').children().off('mouseover', lala)`
You'll see what I explained before.

To do what you tried you should use this : https://api.jquery.com/off/#off-events-selector-handler
And you code will became 
```javascript
$('body').children().on('mouseover', '*', $.noop)
$('body').children().off('mouseover', '*', $.noop)